oauth2.go

```diff
@@ -74,8 +75,9 @@ type TokenSource interface {
 // Endpoint contains the OAuth 2.0 provider's authorization and token
 // endpoint URLs.
 type Endpoint struct {
-	AuthURL  string
-	TokenURL string
+	AuthURL       string
+	DeviceAuthURL string
+	TokenURL      string
```

Comment:
@centimitr these lines are creating a merge conflict, you'd just need to slightly change this bit to match the new version of the file and it'd be a straight merge.